src/main/java/address/MainApp.java

```diff
@@ -2,14 +2,15 @@
 
 import address.model.AddressBook;
 import com.teamdev.jxbrowser.chromium.BrowserCore;
+import com.teamdev.jxbrowser.chromium.LoggerProvider;
```

Comment:
This is to control the jxbrowser logging output, I only used it at the app startup to squash all non-severe messages (see line 54).

`LoggerProvider.setLevel(Level.SEVERE);`

It's not directly related to this PR, should I remove it and push it as a separate commit to master?